crates/cast/src/helpers/constants.rs

```diff
@@ -24,3 +24,4 @@ pub const DEFAULT_RETRIES: u8 = 12;
 pub const DEFAULT_ACCOUNTS_FILE: &str = "~/.starknet_accounts/starknet_open_zeppelin_accounts.json";
 
 pub const KEYSTORE_PASSWORD_ENV_VAR: &str = "KEYSTORE_PASSWORD";
+pub const TEST_KEYSTORE_PASSWORD_ENV_VAR: &str = "TEST_KEYSTORE_PASSWORD";
```

Comment:
why do we need a test variable in cast?

Reply:
Sure, but shouldn't it be declared in tests then? we could allow for variable name manipulation, which would also allow us to remvoe the duplicated code you've linked. Or, we could even set env variable in test and unset it when we're done, if that's more preferable.